pkgs/os-specific/linux/pam/default.nix

```
@@ -2,10 +2,10 @@
 
 stdenv.mkDerivation rec {
   pname = "linux-pam";
-  version = "1.3.1";
+  version = "1.4.0";
 
   src = fetchurl {
-    url    = "https://github.com/linux-pam/linux-pam/releases/download/v1.3.1/Linux-PAM-${version}.tar.xz";
+    url    = "https://github.com/linux-pam/linux-pam/releases/download/v1.4.0/Linux-PAM-${version}.tar.xz";
```

```suggestion
    url    = "https://github.com/linux-pam/linux-pam/releases/download/v{version}/Linux-PAM-${version}.tar.xz";
```

Also why is not fetchFromGitHub used here?

---

Seems like replacing fetchurl by fetchFromGitHub somehow breaks the build.